tests/test-symbols.sh

```diff
@@ -22,7 +22,7 @@ set -xeuo pipefail
 echo '1..3'
 
 released_syms=${G_TEST_SRCDIR}/src/libostree/libostree-released.sym
-if echo "$OSTREE_FEATURES" | grep --quiet --no-messages "DevelBuild"; then
+if echo "$OSTREE_FEATURES" | grep --quiet --no-messages "devel"; then
```

Comment:
Oh duh, and the change still passed because at the time there were no `-devel` symbols.

Can you split out this one too?